types/react-table/index.d.ts

```diff
@@ -75,7 +75,7 @@ export type ColumnGroup<D extends object = {}> =
         })
     )
     // Not used, but needed for backwards compatibility
-    & { accessor?: Accessor<D>; };
+    & { accessor?: keyof D extends never ? IdType<D> | Accessor<D> : Accessor<D>; };
```

Comment:
I'll need to test this, but my guess is that this effectively undoes the `ColumnWithStrictAccessor` changes.

Reply:
@kizu tagging you in the hopes that you might have more time to look at this before I do :)